src/test/java/ru/testtask/service/ProjectServiceTest.java

```diff
@@ -60,4 +73,11 @@ public void checkSimilarNamesCreationTest(){
         Project resultProject = projectService.createProject(testProject);
         Project resultProject2 = projectService.createProject(testProject);
     }
+
+    @Test
+    public void isCurrentUserOwnerOfTest(){
```

Comment:
The test is failed. You need to use project id as parameter of method `projectService.isCurrentUserOwnerOf("a")`.